docs/references/functions.md

```diff
@@ -40,6 +40,7 @@ Available through the _ExpressionConfiguration.StandardFunctionsDictionary_ cons
 | STR_FORMAT(format [,argument, ...]) | Returns a formatted string using the specified format string and arguments, using the configured locale |
 | STR_LOWER(value)                    | Converts the given value to lower case                                                                  |
 | STR_STARTS_WITH(string, substring)  | Returns true if the string starts with the substring (case-sensitive)                                   |
+| STR_TRIM(string)                    | Returns the given string without whitespace                                                             |
```

Comment:
Be more clear in the description, e.g.:
Returns the given string with all leading and trailing space removed.